.gitignore

```
@@ -1,3 +1,6 @@
 /target
 **/*.rs.bk
 /node_modules
+lib
+*.log
+
```

Just to avoid yarn lock files creeping in for now.

```suggestion
yarn.lock

```